src/child/containers/App.js

```diff
@@ -18,13 +18,13 @@ require('script!../../../node_modules/BitFlux/node_modules/bootstrap/js/dropdown
 require('script!../../../node_modules/d3fc/dist/d3fc.bundle.min.js');
 require('script!../../../node_modules/BitFlux/dist/bitflux.js');
 
-const App = ({ code, name }) => (
-    <div className="main">
+const App = ({ code, name, windowState }) => (
+    <div className={`main${windowState.isCompact ? ' compact' : ''}`}>
         <SideBar />
         <div className="main-content">
             <Toolbar />
             <img className={`no-favourites-image${code ? ' hidden' : ''}`} src={noFavourites} />
-            <div className={code ? '' : 'hidden'}>
+            <div className={code && !windowState.isCompact ? '' : 'hidden'}>
```

Comment:
If this needs to be hidden when the app is in compact view, can this be done in the styles? (Not sure if `.right-content` (from main-compact.less) is appropriate? If not, then a new class like `.showcase` or `.chart` could work?)

```less
.compact .someClass {
  display: none;
}
```